### PR TITLE
Add device-file for Logitech MX Anywhere 3B

### PR DESCRIPTION
**Pull request by olmari**
This variant stands for Logitech MX Anywhere 3 for Business.

Signed-off-by: Sami Olmari <sami@olmari.fi>

### CODE REVIEW

**Comment by olmari** [comment]
I took Anywhere 3 device-file and changed ID's to match 3B, seems to work a okay.

**Comment by whot** [comment]
Looking at [the Solaar list of receivers](https://pwr-solaar.github.io/Solaar/devices.html) this is a receiver device, so we can't add it as device file, it would catch all other devices on that same receiver type. I've updated the receiver check in #1314 now but for this to fully work we'll need kernel support. Unfortunately I've paged out everything regarding that, maybe @FFY00 has some memory of it.

**Comment by olmari** [comment]
Well, in mean time adding only bluetooth "side" then, as it works with both bleutooth and the whatever logitech "not bluetooth" dongle it is :)

File: data/devices/logitech-MX-Anywhere3B.device [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Logitech MX Anywhere 3B
+[Device]
+Name=Logitech MX Anywhere 3B
+DeviceMatch=bluetooth:046d:b02d;usb:046d:c548
```
Comment (whot) [inline data/devices/logitech-MX-Anywhere3B.device:4]:
```suggestion
DeviceMatch=bluetooth:046d:b02d
```

**Review by whot — APPROVED** [review]
Going to merge this with the USB match removed, better than having nothing :)

**Pull request merged**